src/NodeGroup/index.d.ts

```diff
@@ -1,17 +1,19 @@
-import * as React from "react";
+import * as React from "react"
 import { HashMap, GetInterpolator } from '..'
 
-export interface INodeGroupProps {
-  data: Array<any>;
-  keyAccessor: (data: any, index: number) => string | number;
-  interpolation?: GetInterpolator;
-  start: (data: any, index: number) => HashMap;
-  enter?: (data: any, index: number) => (HashMap | Array<HashMap>);
-  update?: (data: any, index: number) => (HashMap | Array<HashMap>);
-  leave?: (data: any, index: number) => (HashMap | Array<HashMap>);
-  children: (nodes: Array<any>) => React.ReactElement<any>;
+export interface INodeGroupProps<T> {
+  data: Array<T>
+  keyAccessor: (data: T, index: number) => string | number
+  interpolation?: GetInterpolator
+  start: (data: T, index: number) => HashMap
+  enter?: (data: T, index: number) => HashMap | Array<HashMap>
+  update?: (data: T, index: number) => HashMap | Array<HashMap>
+  leave?: (data: T, index: number) => HashMap | Array<HashMap>
+  children: (nodes: Array<any>) => React.ReactElement<any>
```

Comment:
I feel like it's just as important to tighten-up the `any` type here too. I've got this type in my codebase:

```ts
type ReactMoveNode<Data, State> = {
  key: string;
  data: Data;
  state: State;
};
```

That I'm using like this:

```js
<NodeGroup
 // ...etc
>
  {(
    nodes: Array<ReactMoveNode<{ id: string }, { top: number; opacity: number }>>
  ) => (
    // ...etc
  )}
</NodeGroup>
```

It should be possible to wrap all that up together here.